samples/ui-library/src/components/UILibraryComponent/FormControls.tsx

```diff
@@ -261,7 +261,7 @@ function SelectDemo() {
                         id="demo-simple-select-error"
                         value={age}
                         onChange={handleChange}
-                        renderValue={(value) => `⚠️  - ${value}`}
+                        renderValue={(value) => `⚠️  - ${value as string}`}
```

Comment:
Would this work instead?
```suggestion
                        renderValue={(value: string) => `⚠️  - ${value}`}
```

Reply:
Unfortunately that causes a different error as `value` is typed as `unknown` which forces you to cast.